README.rst

```diff
@@ -24,6 +29,13 @@ When using ``.at`` with a past hour, the command will assume tomorrow instead
 of today: setting a reminder for 9 a.m. when it's 10 a.m. will create a
 reminder for 9 a.m. tomorrow.
 
+You can also use a date instead of a time, or you can use both, placed before
+or after the time::
+
+    .at 2023-06-27 Python 3.7 EOL
+    .at 12:00 2023-06-27 Python 3.7 EOL
+    .at 2023-06-27 12:00 Python 3.7 EOL
```

Comment:
I had to read the code to determine what passing only a date would do, and that shouldn't be necessary.

```suggestion
You can also use a date instead of a time, or you can use both, placed before
or after the time::

    .at 2023-06-27 Python 3.7 EOL
    .at 12:00 2023-06-27 Python 3.7 EOL
    .at 2023-06-27 12:00 Python 3.7 EOL

Passing only a date will set a reminder on that date with *the current time*
(not adjusted for summer/daylight-savings).
```

Reply:
Ha right! I forgot about that case. That's why I wanted a review from you, I knew you'd find these little errors. 😁